src/python/tritonfrontend/CMakeLists.txt

```diff
@@ -153,14 +146,21 @@ if(${TRITON_ENABLE_GPU})
 endif()
 
 if(${TRITON_ENABLE_TRACING})
-  target_include_directories(
+    target_include_directories(
       py-bindings
-      PRIVATE ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+      PUBLIC ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
```

Comment:
I think this may also be unnecessary now for the same reason as described below.

Reply:
Addressed in comment above.